### PR TITLE
dev, v3.0/tidb-in-kubernetes: update tidb-operator-overview

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? 

This PR updates the TiDB Operator Overview according to the latest updates from the docs-cn repository.

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1625
https://github.com/pingcap/docs/pull/1386

### Which version does your change affect? 

dev, v3.0

### Checklist 

- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/tidb-in-kubernetes/tidb-operator-overview.md [hunk 1]
```diff
@@ -12,12 +12,13 @@ TiDB Operator is an automatic operation system for TiDB clusters in Kubernetes.
 
 ![TiDB Operator Overview](/media/tidb-operator-overview.png)
 
-TiDB Operator consists of the following components:
+`TidbCluster` is a custom resource defined by CRD (`CustomResourceDefinition`), which is used to describe the TiDB cluster status expected by the user. The following components are responsible for the orchestration and scheduling logic for a TiDB cluster:
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/tidb-operator-overview.md]:
```suggestion
`TidbCluster` is a custom resource defined by CRD (`CustomResourceDefinition`) and is used to describe the TiDB cluster status expected by the user. The following components are responsible for the orchestration and scheduling logic in a TiDB cluster:
```

File: dev/tidb-in-kubernetes/tidb-operator-overview.md [hunk 2]
```diff
@@ -12,12 +12,13 @@ TiDB Operator is an automatic operation system for TiDB clusters in Kubernetes.
...
 
 * `TidbCluster` is a custom resource defined using CRD (`CustomResourceDefinition`) to describe the desired state of the TiDB cluster;
 * `tidb-controller-manager` is a set of custom controllers in Kubernetes. These controllers constantly compare the desired state recorded in the `TidbCluster` object with the actual state of the TiDB cluster. They adjust the resources in Kubernetes to drive the TiDB cluster to meet the desired state;
 * `tidb-scheduler` is a Kubernetes scheduler extension that injects the TiDB specific scheduling policies to the Kubernetes scheduler;
-* `tkctl` is the command-line interface for TiDB clusters in Kubernetes. It is used for cluster operations and troubleshooting cluster issues.
+
+Besides, TiDB Operator also provides `tkctl`, the command-line interface for TiDB clusters in Kubernetes. It is used for cluster operations and troubleshooting cluster issues.
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/tidb-operator-overview.md]:
```suggestion
In addition, TiDB Operator also provides `tkctl`, the command-line interface for TiDB clusters in Kubernetes. It is used for cluster operations and troubleshooting cluster issues.
```

File: v3.0/tidb-in-kubernetes/tidb-operator-overview.md [hunk 3]
```diff
@@ -12,12 +12,13 @@ TiDB Operator is an automatic operation system for TiDB clusters in Kubernetes.
...
+`TidbCluster` is a custom resource defined by CRD (`CustomResourceDefinition`), which is used to describe the TiDB cluster status expected by the user. The following components are responsible for the orchestration and scheduling logic for a TiDB cluster:
 
 * `TidbCluster` is a custom resource defined using CRD (`CustomResourceDefinition`) to describe the desired state of the TiDB cluster;
 * `tidb-controller-manager` is a set of custom controllers in Kubernetes. These controllers constantly compare the desired state recorded in the `TidbCluster` object with the actual state of the TiDB cluster. They adjust the resources in Kubernetes to drive the TiDB cluster to meet the desired state;
 * `tidb-scheduler` is a Kubernetes scheduler extension that injects the TiDB specific scheduling policies to the Kubernetes scheduler;
-* `tkctl` is the command-line interface for TiDB clusters in Kubernetes. It is used for cluster operations and troubleshooting cluster issues.
+
+Besides, TiDB Operator also provides `tkctl`, the command-line interface for TiDB clusters in Kubernetes. It is used for cluster operations and troubleshooting cluster issues.
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/tidb-operator-overview.md]:
```suggestion
In addition, TiDB Operator also provides `tkctl`, the command-line interface for TiDB clusters in Kubernetes. It is used for cluster operations and troubleshooting cluster issues.
```

File: v3.0/tidb-in-kubernetes/tidb-operator-overview.md [hunk 4]
```diff
@@ -12,12 +12,13 @@ TiDB Operator is an automatic operation system for TiDB clusters in Kubernetes.
 
 ![TiDB Operator Overview](/media/tidb-operator-overview.png)
 
-TiDB Operator consists of the following components:
+`TidbCluster` is a custom resource defined by CRD (`CustomResourceDefinition`), which is used to describe the TiDB cluster status expected by the user. The following components are responsible for the orchestration and scheduling logic for a TiDB cluster:
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/tidb-operator-overview.md]:
```suggestion
`TidbCluster` is a custom resource defined by CRD (`CustomResourceDefinition`) and is used to describe the TiDB cluster status expected by the user. The following components are responsible for the orchestration and scheduling logic in a TiDB cluster:
```

**Comment by TomShawn** [comment]
PTAL again, thanks! @lilin90 @aylei

**Pull request merged**